src/util/pelz_service.c

```diff
@@ -26,6 +29,16 @@ int pelz_service(int max_requests)
   int socket_listen_id;
   pthread_t tid[max_requests];
 
+  int fd;
+  int mode = 0777;              //the file premissions to set rw for all users
+  char buf[25];                 //25 is used because the input line most likely will not be more then 25 characters
```

Comment:
This char buffer will probably need to be longer if it will ever include file paths